### PR TITLE
add info / error banner

### PR DESCRIPTION
**Pull request by DjamelSoualmi**
[ticket](https://trello.com/c/FMP2VyMl/2365-banni%C3%A8res-dinfo-et-derreur)
Information banner:
image

Error banner 
image

- [ ] **Breaking changes ?**  
       If checked, what have you broken ?

- [ ] **Extra lib ?**
      If checked, Which extra lib did you add ? (name, purpose, link ...).

**Testing Strategy**

- [ ] Already covered by tests
- [ ] Manual testing
- [ ] Unit testing

### CODE REVIEW

File: packages/@coorpacademy-components/src/molecule/banner/index.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    </div>
+  );
+};
+
+Banner.propTypes = {
+  type: PropTypes.oneOf('information', 'error'),
+  message: PropTypes.string.isRequired,
+  firstCTA: PropTypes.func,
+  firstCTALabel: PropTypes.string,
+  secondeCTALabel: PropTypes.string,
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/molecule/banner/index.js:42]: second

File: packages/@coorpacademy-components/src/variables/colors.css [hunk 2]
```diff
@@ -39,6 +39,7 @@
 @value cm_positive_100: #35CC7F;
 @value cm_negative_200: #b81400;
 @value cm_orange_200: #FF7043;
+@value cm_negative_50: #ffe5e6;
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/variables/colors.css:42]: tu peux le mettre dans l'ordre, stp

File: packages/@coorpacademy-components/src/variables/colors.css [hunk 3]
```diff
@@ -48,3 +49,5 @@
 @value cm_bleu_article: #365FCD;
 @value cm_red_video: #E8335E;
 @value cm_violet_podcast: #432ba7;
+@value cm_warning_200: #db9200;
+@value cm_warning_50: rgba(255, 192, 67, 0.13);;
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/variables/colors.css:53]: pareil

File: packages/@coorpacademy-components/src/variables/colors.css [hunk 4]
```diff
@@ -39,6 +39,7 @@
 @value cm_positive_100: #35CC7F;
 @value cm_negative_200: #b81400;
 @value cm_orange_200: #FF7043;
+@value cm_negative_50: #ffe5e6;
 @value cm_orange_700: #ff4105;
 @value cm_green_200: #16ac8c;
 @value cm_green_50: #e8fcf8;
```
Comment (jomaora) [inline packages/@coorpacademy-components/src/variables/colors.css:45]: profiter pour trier ça

**Review by jomaora — CHANGES_REQUESTED** [review]

**Review by jomaora — APPROVED** [review]

File: packages/@coorpacademy-components/src/molecule/banner/style.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+    gap: 8px; 
+    padding: 16px 24px; 
+    border-radius: 5px;
+    background-color: cm_warning_50;
+    color: cm_warning_200;
+    border-radius: 5px;
+}
+
+.errorBanner {
+    composes: banner;
```
Comment (esa-coorp) [inline packages/@coorpacademy-components/src/molecule/banner/style.css:25]: pas besoin
`className={classnames(style.banner, type === 'error' && style.errorBanner)}>`
```suggestion
```

**Review by esa-coorp — CHANGES_REQUESTED** [review]

File: packages/@coorpacademy-components/src/molecule/banner/style.css (hunk 5 above)
Comment (DjamelSoualmi) [reply packages/@coorpacademy-components/src/molecule/banner/style.css:25]: aah yes , bien vu

**Review by esa-coorp — APPROVED** [review]

**Pull request merged**